platform/android/build.gradle.template

```diff
@@ -26,9 +26,10 @@ android {
 	lintOptions {
 		abortOnError false
 		disable 'MissingTranslation'
+		//checkReleaseBuilds false
 	}
 
-	compileSdkVersion 23
+	compileSdkVersion 19
```

Comment:
I don't know how to refer multiple lines with github line note.
would you revert it to 23 too?